### ISSUE TITLE
Shader resetting to gray scale on Run or Save

### ISSUE THREAD

**Issue by assimoes** [SebLague/Procedural-Planets#8]
Hey!

I am getting an issue with Unity 2019.x

After generating the planet, the shader works perfectly fine. But, as soon as you save the scene or press Run, the planet goes back to using what seems to be the default material and becomes all grey?

Im attaching the images that show this:

After clicking Generate Planet:

![image](https://user-images.githubusercontent.com/6333324/84884907-363a7e80-b08a-11ea-9111-4de84170e381.png)

After clicking Save (or pressing CTRL + S):

![image](https://user-images.githubusercontent.com/6333324/84884994-52d6b680-b08a-11ea-8b9f-8c0665da3b11.png)

Any hints of why this is happening?

**Comment by firez2469** [comment]
Yes, I had the same problem. What I did was make the generatePlanet() method automatically run on start(). That should prevent you from having to constantly regenerate it.

**Comment by assimoes** [comment]
That makes sense actually. Ill give it a try when I get home.
Thanks man!

A quarta, 17/06/2020, 15:50, firez2469 <notifications@github.com> escreveu:
